source/broadband/webpa_notification.c

```diff
@@ -468,6 +477,14 @@ static void getNotifyParamList(const char ***paramList, int *size)
         }
         count = count-1;
     }
+
+    // Remove Advanced Security params from NotifyList when cujo/fp is not enabled.
+    if(fpFlag == 0)
```

Comment:
Using variable default value as condition check is not a good practice.

Reply:
Incorporated.